tox.ini

```diff
@@ -0,0 +1,37 @@
+[tox]
+envlist =
+    unit
+    unit-buildbot
+    functest
+    functest-buildbot
+
+[testenv:unit]
+basepython = python2.7
+deps = -rrequirements.unittest.txt
+changedir = f5_openstack_agent
+commands = py.test -svvra --cov {posargs}
+
+[testenv:unit-buildbot]
+basepython = python2.7
+changedir = f5_openstack_agent
+deps =
+    git+ssh://git@bldr-git.int.lineratesystems.com/tools/pytest-meta.git
+    git+ssh://git@bldr-git.int.lineratesystems.com/tools/pytest-autolog.git
+    -rrequirements.unittest.txt
+commands = py.test -svvra --cov {posargs}
+
+[testenv:functest]
+basepython = python2.7
+changedir = test/functional/
+deps = -rrequirements.functest.txt
+commands = py.test --cov -svra {posargs}
+
+[testenv:functest-buildbot]
+basepython = python2.7
+changedir = test/functional/
+deps =
+    git+ssh://git@bldr-git.int.lineratesystems.com/tools/pytest-meta.git
+    git+ssh://git@bldr-git.int.lineratesystems.com/tools/pytest-autolog.git
+    git+ssh://git@bldr-git.int.lineratesystems.com/tools/pytest-symbols.git
+    -rrequirements.functest.txt
+commands = py.test --cov -svra {posargs}
```

Comment:
Was there are reason that the verbosity of this py.test command is -svra instead of -svvra?